scout/server/blueprints/variants/templates/variants/cancer-sv-variants.html

```diff
@@ -67,8 +67,8 @@
           <th>Gene(s)</th>
           <th>Region</th>
           <th>Function</th>
-          <th data-bs-toggle="tooltip" data-bs-placement="top" title="Tumor alt. AF. &#013; Alt. allele count | Ref. allele count">Tumor</th>
-          <th data-bs-toggle="tooltip" data-bs-placement="top" title="Normal alt. AF. &#013; Alt. allele count | Ref. allele count">Normal</th>
+          <th data-bs-toggle="tooltip" data-bs-placement="top" title="Tumor alt. AF not computed as it varies with variant type and caller. &#013; Alt. allele count | Ref. allele count">Tumor</th>
```

Comment:
Good! Maybe also change the "count"? I think it will likely be misunderstood without explanation:
```suggestion
          <th data-bs-toggle="tooltip" data-bs-placement="top" title="Tumor alt. AF not computed as it varies with variant type and caller. &#013; Alt. allele support | Ref. allele support">Tumor</th>
```